aws/files/userdata_agent

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash -x
+export curlprefix=appropriate
+export rancher_server_ip='${server_address}'
+
+INSTANCELOCALIP=`curl -s http://169.254.169.254/latest/meta-data/local-ipv4`
+INSTANCEPUBLICIP=`curl -s http://169.254.169.254/latest/meta-data/public-ipv4`
```

Comment:
Both can be retrieved by passing either `awslocal` or `awspublic` as parameter to `--internal-address` and `--address`